src/pixelator/plot/__init__.py

```diff
@@ -47,6 +57,56 @@ def _calculate_densities(coordinates, distance_cutoff, unit_sphere_surface):
     return densities
 
 
+def pxContent_vs_Tau(data: pd.DataFrame, group_by: Optional[str] = None) -> ggplot:
```

Comment:
Interesting way of naming things. Is this common in R?

Reply:
Not really no. We can change the name if you prefer. Any suggestions?

Reply:
I think this answer is good to as why I reacted to this: https://stackoverflow.com/a/8908798

No preference really. Maybe `scatter_umi_per_upia_vs_tau`? This is really what it is.

Let's chat about standarizing names, but using the type of plot seems to be a good idea when it comes to developing a understandable API for the analyst using these functions. What do you think?

Reply:
I'd be interested in getting there the name of the plot type that the user is going to get there. But let's discuss that.

Reply:
Changed name of function to `scatter_umi_per_upia_vs_tau`.